src/components/answer-text-input.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+
+class TextInputAnswerComponent extends Component {
+	constructor(props) {
+		super(props);
+
+		this.state = this.getInitialState();
+	}
+
+	getInitialState = () => {
```

Comment:
Not sure if I have my wires crossed, but since you're in a class, shouldn't your methods be in the `myMethod() { ... }` style rather than `myMethod = () => { ... }`?

Reply:
updated to bind in the constructor.